hat/urls.py

```diff
@@ -17,23 +17,42 @@
 admin.site.site_title = "Iaso"
 admin.site.index_title = "Administration de Iaso"
 
+
 if settings.DISABLE_PASSWORD_LOGINS:
     urlpatterns = [
-        path("login/", TemplateView.as_view(template_name="iaso/disabled_password_login.html"), name="login"),
-        path("admin/login/", TemplateView.as_view(template_name="iaso/disabled_password_login.html"), name="login"),
+        # FIXME workaround to avoid 500 on these routes
+        path(
+            "accounts/admin/login/",
+            auth.views.LoginView.as_view(template_name="iaso/disabled_password_login.html"),
```

Comment:
actually, here, by replacing templateview by loginView, this woul re-enable the login password through a post request